### DISCUSSION TITLE
Setting A/V delay from command arguments

### DISCUSSION THREAD

**Discussion by sogorman** [CESNET/UltraGrid#337] [Q&A]
Apologies if this has been answered but is it possible to set the A/V delay: from the command line when launching UV? I.e. if i press ```<``` a couple of times and arrive at ```Setting A/V delay: -90```, can I launch UG with -90 initially?

**Comment by alatteri** [comment]
on receiver:
 `--audio-delay XX` (ms)

on encoder:
`--audio-filter delay:XX:frames`

**Comment by sogorman** [comment]
Appreciate you!
